lib/src/executables/react16_upgrade.dart

```diff
@@ -28,29 +30,115 @@ const _changesRequiredOutput = """
 Then, review the the changes, address any FIXMEs, and commit.
 """;
 
+String react16CodemodName = 'React16 Codemod';
+
+YamlMap pubspecYaml;
+
+Map<String, List<VersionConstraint>> packagesToCheckFor = {
+  'react': [
+    VersionConstraint.parse('^4.0.0'),
+    VersionConstraint.parse('^5.0.0')
+  ],
+  'over_react': [
+    VersionConstraint.parse('^2.0.0'),
+    VersionConstraint.parse('^3.0.0')
+  ]
+};
+
+VersionConstraint getDependencyVersion(String packageName) {
+  try {
+    if (pubspecYaml.containsKey('dependencies') &&
+        pubspecYaml['dependencies'].containsKey(packageName)) {
+      return VersionConstraint.parse(pubspecYaml['dependencies'][packageName]);
+    } else if (pubspecYaml.containsKey('dev_dependencies') &&
+        pubspecYaml['dev_dependencies'].containsKey(packageName)) {
+      return VersionConstraint.parse(
+          pubspecYaml['dev_dependencies'][packageName]);
+    }
+  } catch (e) {
+    return null;
+  }
+  return null;
+}
+
 void main(List<String> args) {
-  final query = FileQuery.dir(
-    pathFilter: isDartFile,
-    recursive: true,
-  );
-  exitCode = runInteractiveCodemodSequence(
-    query,
-    [
-      ReactDomRenderMigrator(),
-      ReactStyleMapsUpdater(),
-    ],
-    args: args,
-    defaultYes: true,
-    changesRequiredOutput: _changesRequiredOutput,
-  );
-
-  final logger = Logger('over_react_codemod.fixmes');
-  for (var dartFile in query.generateFilePaths()) {
-    final dartSource = File(dartFile).readAsStringSync();
-    if (dartSource.contains('[ ] $manualValidationCommentSubstring')) {
-      logger.severe(
-          'over_react_codemod validation comments are unaddressed in $dartFile');
-      exitCode = 1;
+  Map<String, bool> inTransition = {};
+  bool foundReactOrOverReact = false;
+
+  final react16CodemodLogger = Logger('over_react_codemod.react16');
+  react16CodemodLogger.onRecord.listen((LogRecord record) {
+    print(record.message);
```

Comment:
This should log errors and stacktraces if there are any. Here's a logger impl we have that does this:
https://github.com/Workiva/dart_storybook/blob/master/lib/src/test_utils/runner/utils/logging.dart